doc/source/skips/1-governance.rst

```diff
@@ -0,0 +1,163 @@
+.. _governance:
+
+:Author: Juan Nunez-Iglesias <juan.nunez-iglesias@monash.edu>
+:Author: Stéfan van der Walt <stefanv@berkeley.edu>
+:Author: Josh Warner
+:Author: François Boulogne
+:Author: Emmanuelle Gouillart
+:Author: Mark Harfouche
+:Author: Lars Grüter
+:Author: Egor Panfilov
+:Status: Draft
+:Type: Process
+:Created: 2019-07-02
+:Resolved:
+:Resolution:
+:Version effective: 0.16
+
+===========================================
+scikit-image governance and decision-making
+===========================================
+
+The purpose of this document is to formalize the governance process used by the
+scikit-image project, to clarify how decisions are made and how the various
+elements of our community interact.
+
+This is a consensus-based community project. Anyone with an interest in the
+project can join the community, contribute to the project design, and
+participate in the decision making process. This document describes how that
+participation takes place, how to find consensus, and how deadlocks are
+resolved.
+
+Roles And Responsibilities
+==========================
+
+Contributors
+------------
+Contributors are community members who contribute in concrete ways to the
+project. Anyone can become a contributor, and contributions can take many forms
+– not only code – as detailed in the
+:ref:`contributors guide <howto_contribute>`.
+
+Core developers
+---------------
+Core developers are community members that have demonstrated continued
+commitment to the project through ongoing engagement with the community. They
+have shown they can be trusted to maintain scikit-image with care. Being a core
+developer allows contributors to more easily carry on with their project
```

Comment:
How can we tighten up this sentence?  Does it really allow contributors to more easily carry on with their project related activities?  Not to a large degree.  But it does give them a more definitive say in project decisions, including what code is accepted into the project (via their 1-out-of-2 PR review powers).

Reply:
Maybe moving the last sentence of the paragraph to here would address that point?

Reply:
I've moved it, see what you think now...